contracts/token/psp34/psp34.rs

```diff
@@ -1,13 +1,13 @@
-pub use crate::traits::psp721::*;
+pub use crate::traits::psp34::*;
 use brush::{
     declare_storage_trait,
     traits::{
         AccountId,
         Flush,
     },
 };
-pub use derive::PSP721Storage;
-use ink_env::Error as EnvError;
+pub use derive::PSP34Storage;
+use ink_env::{Error as EnvError};
```

Comment:
Did you apply `cargo fmt`?

Reply:
No, i didn't.
I thought it was applied on save, but turns out it wasn't.